### DISCUSSION TITLE
关于.txt文件中数据格式的问题

### DISCUSSION THREAD

**Discussion by tang-mouren** [liusy58/NJU_AdvancedProgramming_SP22#21] [Q&A] [answered]
一定要按照要求的那样，写成  U001,南大,123456,12345678,仙林大道163号,1024.0,正常  一组数据中多个子数据用逗号这样的格式来分隔开吗？可不可以使用空格来分离？如果一定要用逗号来分离的话，请问加入用户在用户名，地址等方面的数据中输入了逗号，这样会不会影响解析呢？

**Comment by binbin00** [comment, marked as answer]
一定要按要求写，ppt明确要求了用户名和地址等只能包含中文汉字或英文字母。
